src/renderer/components/analytics/AnalyticsComponentRenderer.tsx

```
@@ -0,0 +1,853 @@
+    return <NotImplementedPlaceholder tag={tag} />;
+  }
+
+  const subProps: ChartSubProps = { data, chartProps: parsedProps };
+  const renderNestedContent = (nestedContent?: string): React.ReactNode => {
+    if (!nestedContent?.trim()) return null;
+    const blocks = parseAnalyticsMarkdown(nestedContent);
+    return blocks.map((block, index) => {
+      if (block.type === 'text') {
+        return (
+          <Typography
+            key={`${block.lineStart}-${index}`}
```

Biome (2.5.0)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)